src/ui_framework/doc_site/src/components/guide_example/guide_example.jsx

```diff
@@ -0,0 +1,77 @@
+
+import React, {
+  Component,
+  PropTypes,
+} from 'react';
+
+import {
+  Slugify,
+} from '../../services';
+
+import {
+  GuidePage,
+  GuidePageSection,
+} from '../';
+
+export default class GuideExample extends Component {
+
+  constructor(props, sections) {
```

Comment:
Seeing a second argument to the constructor of a component perplexed me at first. I wonder if it would be less confusing if this class was defined inside the `createExample` function, and the closure was used to give that example component it's `sections`/`examples`.

```js
// I also think createExample could use a bit more color in it's name
export function createExampleComponent(sections) {
  return class ExampleCompoent extends Component {
    // ...
  }
}
```